tests/benchdnn/dnn_types.cpp

```diff
@@ -187,6 +189,7 @@ int attr_t::get_default_mask(policy_t policy, int ndims) {
         case PER_OC:
         case PER_DIM_1: return (1 << 1);
         case PER_OCIC:
+        case MX:
```

Comment:
I think this should use `PER_TENSOR` policy instead to cover 3D, 4D and 5D matmul tensors as well.